### PR TITLE
Console EMAIL_BACKEND added for development purpose

### PR DESCRIPTION
**Pull request by DESHRAJ**

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -176,7 +176,13 @@
 LOGIN_REDIRECT_URL = '/'
 
 # E-Mail Settings
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+
+#For development
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```
Comment (sivaa) [inline settings/common.py]: It should be added to dev.sample.py

**Comment by vnbang2003** [comment]
Can we please do one round of full test . Creating local user and confirming users.
I think we are missing confirmation link .

**Comment by DESHRAJ** [comment]
I have already tested the use-cases and its working fine. The conformation email is sent properly with a confirmation link.

**Comment by vnbang2003** [comment]
@DESHRAJ Can you click on confirm email link it me below error
NoReverseMatch at /accounts/confirm-email/szmusae2dcbhn0omxfmqeorwbtrieg2ktstdgeobjlxwtpb0p3f2rtfdcvfj71mh/

Reverse for 'login' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []

Request Method: 	POST
Request URL: 	http://localhost:8000/accounts/confirm-email/szmusae2dcbhn0omxfmqeorwbtrieg2ktstdgeobjlxwtpb0p3f2rtfdcvfj71mh/

**Comment by mascot6699** [comment]
@DESHRAJ You might need to set LOGIN_REDIRECT_URL to "/account/login/" in settings/common.py to make it work probably

**Comment by DESHRAJ** [comment]
Actually LOGIN_URL = '/account/login' to get this working. Just changing it. 
Thanks for reporting.

**Comment by vnbang2003** [comment]
@DESHRAJ Can you resolve conflicts .
Do you mind to share test case you tried .

**Comment by DESHRAJ** [comment]
Ok sure. I am fixing the conflicts. For the test cases, I haven't written test cases as of now.

**Comment by ultimatecoder** [comment]
@vnbang2003  Requesting to merge bug fixes fast, which smooths new development. Thanks.

**Comment by vnbang2003** [comment]
@ultimatecoder I am merge everyday evening which is good pace to merge.

**Comment by vnbang2003** [comment]
@DESHRAJ What i mean by test case to explain scenarios you tested locally.
  Once we get Test layer integrated we can use them to write code for testcase.

**Pull request merged**